simpledotorg/simple-server
# Add Blood sugar measurement details table

**Story card:** [sc-7784](https://app.shortcut.com/simpledotorg/story/7784/add-bs-measurement-details-card)

## Because

We reverted the previous PR that added the BS measurement details table. 

The previous PR had a bug when a region did not have diabetic data, which would throw an uncaught exception. 

This PR fixes this bug, along with some enhancements to the table, so it matches the tables in hypertension reports.

## This addresses

- Adds Blood sugar measurement table
- Fixes bug to handle regions where diabetic data might be missing
- Updates the table to match tables in hypertension reports

## Test instructions

- Turn on diabetes_management_reports on the flipper
- Visit the reports page of a facility which has enable_diabetes_management set to true
- Click on the Diabetes link
- You should see the BS measure details graph

![Screenshot from 2022-05-19 15-27-08](https://user-images.githubusercontent.com/3933133/169267116-51a28d3f-defb-42c4-9e2b-99e3034fc7d8.png)
